lib/containers/urls.pm

```diff
@@ -293,6 +293,16 @@ sub get_suse_container_urls {
     return (\@untested_images, \@released_images);
 }
 
+# Get single image from CONTAINER_IMAGE_TO_TEST or untested image from get_suse_container_urls()
+sub get_container_image_to_test {
+    my $images_to_test;
```

Comment:
Could we do the following?
1) Remove `get_container_url_from_var` method
2) Write this method like this

```
my $image = get_var('CONTAINER_IMAGE_TO_TEST');
return $image if ($image);
($image, undef) = get_suse_container_urls();
return $image->[0]; # or similar, you can use different var if you want
```

3) For image.pm see below

Reply:
Yes, can you please create a ticket out of this and close?

Reply:
I think [this ticket](https://progress.opensuse.org/issues/108058) is suitable.